charm-slurm-configurator/src/interface_prolog_epilog.py

```diff
@@ -70,12 +70,10 @@ def _on_relation_changed(self, event):
             'slurmctld_prolog_path': prolog
         })
 
-        # Store the license path in the charm's state
-        self._charm.set_slurm_license_available(True)
         self.on.prolog_epilog_available.emit()
 
     def _on_relation_broken(self, event):
-        self._charm.set_slurm_license_available(False)
+        self._stored.prolog_epilog = None
```

Comment:
@ducks23 we should try and be consistent with our typing here. `self._stored.prolog_epilog` is initialized to a dict in `__init__()`, set to a string in `relation_changed`, and set to `NoneType` in `broken`. As part of this pr, will you ensure that the type matches all the way across.

Reply:
it is set to a dict in relation changed